src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -76,33 +77,49 @@ public String getUrlForWiki(boolean addWWW) {
   }
 
   public String getUrlForWiki(String wikiName, boolean addWWW) {
-    EnvType env = Configuration.getEnvType(this.env);
+    EnvType envType = Configuration.getEnvType(this.env);
     final String wikiaName = getWikiaGlobalName(wikiName);
 
     String www = "";
     if (addWWW) {
       www = "www.";
     }
 
-    switch (env) {
+    switch (envType) {
       case DEV: {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested code block into a method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1199)